makefile

```diff
@@ -1,10 +1,7 @@
 MAKEFLAGS += --silent
 
-verify:
-	./scripts/verify.sh
```

Comment:
Is this on purpose?

Reply:
yup. we actually don't have any tests for this repo. we need to add some